doc/design/from_ngsi_events_to_flume_events.md

```diff
@@ -79,12 +82,18 @@ Flume events are not much more different than the above representation: there is
 * The time-to-live (or <b>ttl</b>) specifies the number of re-injection retries in the channel when something goes wrong while persisting the data. This re-injection mechanism is part of the reliability features of Flume. -1 means inifinite retries.
-* The <b>destination</b> headers is used to identify the persistence element within the used storage, i.e. a file in HDFS, a MySQL table or a CKAN resource. This is added by a custom interceptor called `DestinationExtractor` added to the Flume's suite. See the <i>doc/design/interceptors</i> document for more details.
+* Note that Orion can include a `Fiware-Service` HTTP header specifying the tenant/organization associated to the notification. Since version 0.3, Cygnus is able to support this header, although the actual processing of such tenant/organization depends on the particular sink. If the notification doesn't include this header, then Cygnus will use the default service specified in the `default_service` configuration property of `OrionRESTHandler`. Please observe the notified `Fiware-Service` is transformed following the rules described at [`doc/design/naming_conventions.md`](doc/design/naming_conventions.md). This NGSI header is used for building this header:
+    * It is directly added as the `notified-service`. 
+* Orion can notify another HTTP header, `Fiware-ServicePath` specifying a subservice within a tenant/organization. Since version 0.6, Cygnus is able to support this header, although the actual processing of such subservice depends on the particular sink. If the notification doesn't include this header, then Cygnus will use the default service path specified in the `default_service_path` configuration property of `OrionRESTHandler`. Please observe the notified `Fiware-ServicePath` is transformed following the rules described at [`doc/design/naming_conventions.md`](doc/design/naming_conventions.md). This NGSI header is used for building several Flume headers:
+    * It is directly added as the `notified-servicepath`.
+    * It is replicated, per each notified context element, as the `default-servicespaths` array. This is used when the grouping feature is not enabled in the processing sink.
+    * It may also appear in the `grouped-servicepaths` array when, being enabled the grouping feature, there is no rule changing the default servicePath.
+* By the default, the final persistece element (file, table, collection, etc), also named as the <i>destination</i>, is composed as the concatenation of the entity ID and type. This can be changed by the grouping feature, if enabled, deciding a new detination per each notified context element. Thus, the following headers may appear in a Flume event:
+    * `default-destinations`, an array that can be used by those sinks not enabling the grouping feature.
+    * `grouped-destinations`, an array that can be used by those sinks enabling the grouping feature.
 
```

Comment:
This is already included, please check the `ngsi-event` structure starting at line 6 and the `flume-event` structure starting at line 51. In addition, from line 81 there is a detailed explanation on the `flume-event` headers.